implementations/rust/ockam/ockam/src/remote_forwarder.rs

```diff
@@ -1,7 +1,10 @@
 #![deny(missing_docs)]
 
 use crate::{route, Context, OckamError};
+#[cfg(not(feature = "std"))]
+use ockam_core::compat::rand::random;
```

Comment:
Can we simplify this somehow? This looks to me like we need to use this complicated 4-lines random import all the time, am I wrong?

Reply:
`rand` is somewhat of a special case. 

What I originally wanted was to just move it into `compat::` for both `std` and`no_std` builds.

Unfortunately I ran into an issue where different versions of the `rand` crate are being pulled in by the various crates. Specifically there were some breaking changes when I tried to unify the versions.

So this is very much a workaround for now until we can unify the different `rand` versions across the crates. :-P

Reply:
Hm. I think unifying rand version across dependencies may be impossible, unless we want to fork and maintain all of them) There is no guarantee authors will update their crates' rand dependencies in reasonable time, which will make us use the old version of rand, which is IMHO pretty bad. Any other thoughts how we could fix this issue?